test/test_cookbook_geometry.jl

```diff
@@ -502,124 +543,139 @@ end
+        @test AG.getx(point, 0) ≈ x
+        @test AG.gety(point, 0) ≈ y
     end
 
     # Method 3
     point = AG.fromJSON(geojson)
-    println((AG.getx(point, 0), AG.gety(point, 0)))
+    @test AG.getx(point, 0) ≈ x
+    @test AG.gety(point, 0) ≈ y
 end
 
 @testset "Create Geometry from GML" begin
     gml = """<gml:Point xmlns:gml="http://www.opengis.net/gml">
              <gml:coordinates>108420.33,753808.59</gml:coordinates>
              </gml:Point>"""
+    x, y = 108420.33, 753808.59
 
     # Method 1
     point = AG.unsafe_fromGML(gml)
-        println((AG.getx(point, 0), AG.gety(point, 0)))
+    @test AG.getx(point, 0) ≈ x
+    @test AG.gety(point, 0) ≈ y
     AG.destroy(point)
 
     # Method 2
     AG.fromGML(gml) do point
-        println((AG.getx(point, 0), AG.gety(point, 0)))
+        @test AG.getx(point, 0) ≈ x
+        @test AG.gety(point, 0) ≈ y
     end
 
     # Method 3
     point = AG.fromGML(gml)
-    println((AG.getx(point, 0), AG.gety(point, 0)))
+    @test AG.getx(point, 0) ≈ x
+    @test AG.gety(point, 0) ≈ y
 end
 
 @testset "Create Geometry from WKB" begin
     wkb = [0x01,0x01,0x00,0x00,0x00,0x7b,0x14,0xae,0x47,0x45,0x78,0xfa,0x40,
            0xe1,0x7a,0x14,0x2e,0x21,0x01,0x27,0x41]
+    x, y = 108420.33, 753808.59
 
     # Method 1
     point = AG.unsafe_fromWKB(wkb)
-        println((AG.getx(point, 0), AG.gety(point, 0)))
+    @test AG.getx(point, 0) ≈ x
+    @test AG.gety(point, 0) ≈ y
     AG.destroy(point)
 
     # Method 2
     AG.fromWKB(wkb) do point
-        println((AG.getx(point, 0), AG.gety(point, 0)))
+        @test AG.getx(point, 0) ≈ x
+        @test AG.gety(point, 0) ≈ y
     end
 
     # Method 3
     point = AG.fromWKB(wkb)
-    println((AG.getx(point, 0), AG.gety(point, 0)))
+    @test AG.getx(point, 0) ≈ x
+    @test AG.gety(point, 0) ≈ y
 end
 
-@testset "Count Points in a Geometry" begin
+@testset "Count Points in a LineString" begin
     wkt = "LINESTRING (1181866.263593049 615654.4222507705, 1205917.1207499576 623979.7189589312, 1227192.8790041457 643405.4112779726, 1224880.2965852122 665143.6860159477)"
 
     # Method 1
     AG.fromWKT(wkt) do geom
-        println("Geometry has $(AG.npoint(geom)) points")
+        @test AG.npoint(geom) == 4
     end
 
     # Method 2
-    println("Geometry has $(AG.npoint(AG.fromWKT(wkt))) points")
+    @test AG.npoint(AG.fromWKT(wkt)) == 4
 end
 
-@testset "Count Geometries in a Geometry" begin
+@testset "Count Points in a MultiPoint" begin
     wkt = "MULTIPOINT (1181866.263593049 615654.4222507705, 1205917.1207499576 623979.7189589312, 1227192.8790041457 643405.4112779726, 1224880.2965852122 665143.6860159477)"
 
     # Method 1
     AG.fromWKT(wkt) do geom
-        println("Geometry has $(AG.npoint(geom)) points")
+        # TODO This should be 4!
+        @test AG.npoint(geom) == 0
     end
 
     # Method 2
-    println("Geometry has $(AG.npoint(AG.fromWKT(wkt))) points")
+    # TODO This should be 4!
+    @test AG.npoint(AG.fromWKT(wkt)) == 0
```

Comment:
Should npoint return 4 here?

Reply:
yeah it should! what's the reason why it's returning 0 here?

Reply:
I don't know, seems a bug to me.

Reply:
Created #30 for this, as tests are now failing on this.

Reply:
My recommendation is to keep it at 0 for now, and introducing a new test for `AG.ngeom` here to demonstrate the contrast in behavior, while we debate about what's the right way to go at #30

Reply:
Agreed.